internal/cluster/cluster.go

```diff
@@ -42,9 +44,27 @@ func NewFullCluster(authApi ybmAuthClient.AuthApiClient, clusterData ybmclient.C
 	fc.SetVPCs(authApi)
 	fc.SetAllowLists(authApi)
 	fc.SetNodes(authApi)
+	fc.SetCMK(authApi)
 	return fc
 }
 
+func (f *FullCluster) SetCMK(authApi ybmAuthClient.AuthApiClient) {
+	resp, r, err := authApi.ListClusterCMKs(f.Cluster.Info.Id).Execute()
+	if err != nil {
+		if err.Error() == "409 Conflict" {
```

Comment:
On the UI I was never getting 409 but 404 which is strange, maybe this has been fixed.

Reply:
I believe it depends on the API. If the API requires a cluster to be healthy, it should return 409.